README.md

```diff
@@ -36,25 +36,62 @@ The repository contains native code project required for the Windows installer.
 - To build in VS, run a command line build first, then run `artifacts\core-sdk-build-env.bat` from a VS command prompt and then `devenv Microsoft.DotNet.Cli.sln`
 - To test different languages of the installer, run `artifacts\packages\Debug\Shipping>dotnet-sdk-3.1.412-win-x64.exe /lang 1046` using the LCID of the language you want to test
 
-# Building (source-build)
+# Build .NET from source (source-build)
```

Comment:
Should we include the support policy section here?  There have been more and more folks trying to build 6.0.3xx which is not supported.

Reply:
Yes, that sounds good to me, I'll make the change.